doc/nrf/device_guides/working_with_nrf/nrf53/features_nrf53.rst

```diff
@@ -0,0 +1,78 @@
+.. _nrf53_features:
+
+Features of nRF53 Series
+########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF53 Series System-on-Chips (SoC) integrate a high-performance Arm® Cortex®-M33 dual-core processor with a 2.4 GHz RF transceiver, in addition to advanced security features.
+All nRF53 Series SoC support Bluetooth® 5.1 and Bluetooth Mesh, in addition to multiprotocol capabilities.
+
+For additional information, refer to the following resources:
+
+* `nRF53 Series`_ for the technical documentation on the nRF53 Series chips and associated kits.
+* :ref:`ug_nrf5340_gs` for getting started with the nRF5340 DK.
+* :ref:`ug_nrf5340` for advanced topics specific to the nRF5340 DK.
+* :ref:`installation` and :ref:`configuration_and_build` documentation to install the |NCS| and learn more about its development environment.
+
+For the Nordic Thingy:53, see the following user guides:
+
+* :ref:`ug_thingy53_gs` getting started with the Thingy:53
+* :ref:`ug_thingy53` for advanced topics specific to the Thingy:53
+
+Dual-core architecture
+**********************
+
+The nRF53 Series introduces a dual-core Arm Cortex-M33 architecture, consisting of the following cores:
+
+* An application core for high-performance application execution.
+* A dedicated network core optimized for wireless communication efficiency.
+
+The dual-core architecture allows for concurrent execution of application logic and wireless communication tasks.
+This ensures application processes do not interfere with network operations, enabling IoT with higher responsiveness and reliability.
+With one core dedicated to application tasks and the other to connectivity, devices can benefit from optimized performance in both areas.
+
+The separation of application and network functionalities simplifies firmware updates, allowing modifications on one processor without affecting the other.
+
+This architecture is ideal for devices requiring significant data processing alongside continuous wireless connectivity.
+
+Inter-core communication
+========================
+
+Interprocessor Communication (IPC) allows for :ref:`ug_nrf5340_intro_inter_core` between the two cores.
```

Comment:
This link seems off to me. The general page for the Series should not have a link that implies the page for one of the devices has the required information.

Not sure how to improve that, though.

Reply:
Maybe that ref can be removed, but text stays the same - since it is linking to the Developing with nRF5340 DK guide?